doc/en/components/multi-column-combobox.md

```diff
@@ -8,25 +8,24 @@ mentionedTypes: []
 
 The Multi-Column Combo Box automatically generates columns for properties on the data object. This component is unique in that it's a combo box that visualizes large amounts of data similar to a data grid embedded in the dropdown. 
 
-Data binding can be achieved using an array of complex objects via the column's `DataSource` property.
+## Usage
 
-The `TextField` property determines which value is shown when users make a selection. 
+The following properties are the most commonly used to configure the component:
 
-The `ValueField` property determines the bound value of the underlying data item selected. This is necessary if your list of objects have several properties.
+- `DataSource` - Data binding can be achieved using an array of complex objects via the column's `DataSource` property.
```

Comment:
change description to:
allows binding data in form of an array of complex objects that will be displayed in drop down menu

Reply:
done